RUFAS/routines/field/crop/root_development.py

```diff
@@ -42,7 +43,7 @@ def develop_roots(self) -> None:
         self.data.root_fraction = self._determine_root_fraction(self.data.heat_fraction)
 
         # update root depth
-        if self.data.is_perennial:
+        if self.data.is_perennial and time.current_calendar_year != self.data.planting_year:
             self.data.root_depth = self.data.max_root_depth  # Note: assumption of SWAT
```

Comment:
```suggestion
            self.data.root_depth = self.data.max_root_depth  
```
Move to notes section.